released/packages/coq-jsast/coq-jsast.2.0.0/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+name: "coq-jsast"
+version: "2.0.0"
+synopsis: "A minimal JavaScript syntax tree carved out of the JsCert project"
+description: """
+A minimal JavaScript syntax tree carved out of the JsCert project, with additional support for let bindings and using native floats.
+"""
+
+maintainer: "Jerome Simeon <jeromesimeon@me.com>"
+authors: [
+    "Martin Bodin <>"
+    "Arthur Charguéraud <>"
+    "Daniele Filaretti <>"
+    "Philippa Gardner <>"
+    "Sergio Maffeis <>"
+    "Daiva Naudziuniene <>"
+    "Alan Schmitt <>"
+    "Gareth Smith <>"
+    "Josh Auerbach <>"
+    "Martin Hirzel <>"
+    "Louis Mandel <>"
+    "Avi Shinnar <>"
+    "Jerome Simeon <>"
+]
+
+license: "BSD-2-Clause"
+homepage: "https://github.com/querycert/jsast"
+bug-reports: "https://github.com/querycert/jsast/issues"
+dev-repo: "git+https://github.com/querycert/jsast/tree/JsAst"
+
+build: [
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+remove: ["rm" "-R" "%{lib}%/coq/user-contrib/JsAst"]
```

Comment:
```suggestion
```
Redundant line.

Reply:
Removed that line, but I'm unclear about the rules here. We had a similar line in this package: https://github.com/querycert/opam-coq-archive/blob/06f9f294f3ae8378b4dfbdaade628c123054ac20/released/packages/coq-qcert/coq-qcert.2.0.0/opam#L25

Is that also redundant?

Reply:
Yes, opam 2 handles the removal automatically by tracking the installed files. This did not exist with opam 1.

Reply:
> Yes, opam 2 handles the removal automatically by tracking the installed files. This did not exist with opam 1.

Thanks for clarifying. Unless you want me to I won't change the `coq-qcert` opam file, but will simply remove that line in future versions.